### ISSUE TITLE
 PerformanceResourceTiming entries do not have size metrics

### ISSUE THREAD

**Issue by harunurhan** [cypress-io/cypress#25224]
### Current behavior

When browser (chrome) is launched and controlled by cypress, [PerformanceResourceTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming) entries does not have size metrics (`decodedBodySize` and `encodedBodySize`), they are all 0.

It could be related to some of the chrome launch flags that cypress is passing, I tried to remove a few but it was still the same.

Visiting the same webpage on a different chrome instance works as expected, that's why I think this could be related to cypress.

### Desired behavior

Metrics should be reported as stated in the docs, only 0 when it's cross-origin (unless [Timing-Allow-Origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Timing-Allow-Origin) is present).

### Test code to reproduce

Visit the webpage (`http://localhost:<port>/<pathname>`) and access/log `performance.getEntriesByType('resource')` either from devtools console or via `cy.window()`

Note that metrics are 0, when page is visited from a cypress test, as well as in an new tab on the same browser instance.

### Cypress Version

11.0.0

### Node version

v16.18.0

### Operating System

macOS 12.5.1

**Comment by lmiller1990** [comment]
I reproduced this:

https://github.com/cypress-io/cypress-test-tiny/compare/master...lmiller1990:cypress-test-tiny:issue-25224?expand=1

## Browser

![Image](https://user-images.githubusercontent.com/19196536/209266654-3cf20cf9-2f51-4e5a-8665-88c6a0b22d3d.png)

## Cypress

No console.log for performance, empty array for the array


![Image](https://user-images.githubusercontent.com/19196536/209266702-07dbf226-b9e0-410e-8f2e-00334486c839.png)

**Comment by lmiller1990** [comment]
Interestingly enough we do have tests that assert against `window.performance.getEntries()`: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/e2e/commands/clock.cy.js#L201

**Comment by harunurhan** [comment]
@lmiller1990 Just to be clear I do see entries on cypress too, but the problem is size properties (`decodedBodySize`, `encodedBodySize` and `transferSize`) are reported as `0` whereas without cypress, all have the correct size.

**Comment by lmiller1990** [comment]
Yeah, I'm not too sure what's going on. I wonder if this API works differently in an iframe... 

As an aside, what's the goal here - are you using Cypress to benchmark your app?

**Comment by harunurhan** [comment]
@lmiller1990, This API works differently for iframe indeed, but I don't think this is an iframe issue because it also happens when I visit my app directly (without cypress runner UI) on the browser that cypress launches. Could one of the launch args that cypress is setting or missing be causing this problem?

> what's the goal here - are you using Cypress to benchmark your app?

Yes the idea is to benchmark the application, I'd like to continuously record/monitor performance metrics (such as loaded assets size per page) as a side-effect for the test runs.

**Comment by wangcheng** [comment]
We are also using cypress to benchmark our app and we ran into the similar situation recently after we upgrade the Chrome version to 107. I found that the metric is correct before (including) Chrome `107.0.5286.0`.

Our cypress version is `9.6.1`. node version is `16.10.0`.

I checked the change log between `107.0.5286.0` and `107.0.5288.0` and found [a suspicious change](https://chromium-review.googlesource.com/c/chromium/src/+/3873286). However I don't have enough understanding of the code to confirm this is the culprit.

**Comment by wangcheng** [comment]
Hi @harunurhan. Did you set `chromeWebSecurity` to `false`? We used to have the same issue. I tried set it to `chromeWebSecurity` to `true` and the issue was gone.
